src/bower_components/emby-webcomponents/globalize.js

```diff
@@ -51,43 +47,37 @@ define(['connectionManager', 'userSettings', 'events'], function (connectionMana
         try {
             dateTimeCulture = userSettings.dateTimeLocale();
         } catch (err) {
-
+            console.log('no date format set in user settings');
         }
 
         if (dateTimeCulture) {
             currentDateTimeCulture = normalizeLocaleName(dateTimeCulture);
-        }
-        else {
+        } else {
             currentDateTimeCulture = currentCulture;
         }
-
         ensureTranslations(currentCulture);
     }
 
     function ensureTranslations(culture) {
-
         for (var i in allTranslations) {
             ensureTranslation(allTranslations[i], culture);
         }
     }
 
     function ensureTranslation(translationInfo, culture) {
-
         if (translationInfo.dictionaries[culture]) {
             return Promise.resolve();
         }
 
         return loadTranslation(translationInfo.translations, culture).then(function (dictionary) {
-
             translationInfo.dictionaries[culture] = dictionary;
         });
     }
 
     function normalizeLocaleName(culture) {
```

Comment:
Why is the normalization needed? And why only for 3 languages?

Reply:
I would rather we stick to standard language codes, so far what I saw seems like a mumbo-jumbo of standard ones and invented by Emby...

Reply:
I was going to leave this for another day, it could break settings that rely on the current language codes. Up to you guys though, would be easy to just remove it and properly name the language files.

Reply:
Can you just drop a `TODO` comment there, so we won't forget? Or open a tracking issue, up to you, just something so we won't lose it forever.